src/core/src/op/paged_attention.cpp

```diff
@@ -147,6 +147,35 @@ void PagedAttentionExtension::validate_and_infer_types() {
                           get_input_element_type(12),
                           ".");
 
+    NODE_VALIDATION_CHECK(
+            this,
+            get_input_partial_shape(13).rank().is_dynamic() ||
+            get_input_partial_shape(13).rank().get_length() == 0 ||
+            get_input_partial_shape(13).rank().get_length() == 1,
+            "Input `rotation_coefficients` should either have an empty shape or rank 1, but it has rank ",
```

Comment:
```suggestion
            "Input `rotation_coefficients` should either have rank 1 or omitted, but it has rank ",
```
"Empty" shape means `[0]` here, which have rank 1.

Reply:
Done